### ISSUE TITLE
Deploying to Elastic Beanstalk?

### ISSUE THREAD

**Issue by marclar** [calebmer/postgraphql#97]
I'm trying to figure out how to start the server from a node.js script (as opposed to using the `postgraphql` command.

Assuming I have `postgraphql` in my package.json, is it possible to create a simple `server.js` file that will start the PostGraphQL server running?

**Comment by calebmer** [comment]
Have you seen the [library documentation article](https://github.com/calebmer/postgraphql/blob/master/docs/library.md)? (I really need to add a link to this in the `README`…)

**Comment by calebmer** [comment]
You can also look at [`main.js`](https://github.com/calebmer/postgraphql/blob/master/src/main.js) (especially near the bottom) to see how we currently launch PostGraphQL from the command line.

**Comment by marclar** [comment]
Hm. Thanks for the fast response, @calebmer . For some reason, it's not working -- I have a [very simple `server.js`](https://gist.github.com/anonymous/1a8112a73158530547533355823969ee), and I keep getting these log messages:

```
-------------------------------------
/var/log/nodejs/nodejs.log
-------------------------------------
    at Function.Module._resolveFilename (module.js:325:15)
    at Function.Module._load (module.js:276:25)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (/var/app/current/node_modules/postgraphql/dist/createServer.js:31:21)
    at Module._compile (module.js:409:26)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (/var/app/current/node_modules/postgraphql/dist/postgraphql.js:13:21)
    at Module._compile (module.js:409:26)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
module.js:327
    throw err;
    ^
```

**Comment by calebmer** [comment]
That would be a bug in PostGraphQL. Fixed in `1.8.3`. My bad :blush:

**Comment by marclar** [comment]
Awesome -- got it working. Thanks!
